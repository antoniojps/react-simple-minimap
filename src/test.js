import React from 'react'
import { render, cleanup } from 'react-testing-library'
import Minimap from './'

afterEach(cleanup)

test('<Minimap>', () => {
  const { container, debug } = render(
    <Minimap
      of={
        <div
          styles={{ width: '100%', height: '2000px', backgroundColor: 'red' }}
        />
      }
    />
  )
  expect(container.firstChild).toMatchSnapshot()
  debug()
})
