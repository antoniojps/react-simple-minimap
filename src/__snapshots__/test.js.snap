// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Minimap> 1`] = `
<div
  style="width: 114px; height: 300px;"
>
  <div>
    <div
      styles="[object Object]"
    />
  </div>
</div>
`;
